boards/sltb001a/include/periph_conf.h

```diff
@@ -0,0 +1,235 @@
+/** @} */
+
+/**
+ * @name    PWM configuration
+ * @{
+ */
+static const pwm_chan_conf_t pwm_channel_config[] = {
+    0                               /* no available channels */
+};
+
+static const pwm_conf_t pwm_config[] = {
+    0                               /* no available timers */
+};
+
+#define PWM_NUMOF           (0U)
+/** @} */
+
+/**
+ * @brief   RTC configuration
+ */
+#define RTC_NUMOF           (1U)
+
+/**
+ * @name    RTT configuration
+ * @{
+ */
+#define RTT_NUMOF           (1U)
+
+#define RTT_MAX_VALUE       (0xFFFFFFFF)
+#define RTT_FREQUENCY       (1U)
+/** @} */
+
+/**
+ * @name    SPI configuration
+ * @{
+ */
+static const spi_dev_t spi_config[] = {
+    {
+        USART1,                             /* device */
+        GPIO_PIN(PC, 6),                    /* MOSI pin */
+        GPIO_PIN(PC, 7),                    /* MISO pin */
+        GPIO_PIN(PC, 8),                    /* CLK pin */
+        USART_ROUTELOC0_RXLOC_LOC11 |
+            USART_ROUTELOC0_TXLOC_LOC11 |
+            USART_ROUTELOC0_CLKLOC_LOC11,   /* AF location */
+        cmuClock_USART1,                    /* CMU register */
+        USART1_RX_IRQn                      /* IRQ base channel */
+    }
+};
+
+#define SPI_NUMOF           (1U)
+#define SPI_0_EN            (1)
+/** @} */
+
+/**
+ * @name    Timer configuration
+ * @{
+ */
+static const timer_conf_t timer_config[] = {
+    {
+        {
+            TIMER0,             /* lower numbered timer */
+            cmuClock_TIMER0     /* pre-scaler bit in the CMU register */
+        },
+        {
+            TIMER1,             /* higher numbered timer, this is the one */
+            cmuClock_TIMER1     /* pre-scaler bit in the CMU register */
+        },
+        TIMER1_IRQn             /* IRQn of the higher numbered timer */
+    }
+};
+
+#define TIMER_NUMOF         (1U)
+#define TIMER_0_EN          (1)
+#define TIMER_0_ISR         isr_timer1
+#define TIMER_0_MAX_VALUE   (0xffff)
+/** @} */
+
+/**
+ * @name    UART configuration
+ * @{
+ */
+static const uart_conf_t uart_config[] = {
+    {
+        USART0,                             /* device */
+        GPIO_PIN(PA, 1),                    /* RX pin */
+        GPIO_PIN(PA, 0),                    /* TX pin */
+        USART_ROUTELOC0_RXLOC_LOC0 |
+            USART_ROUTELOC0_TXLOC_LOC0,     /* AF location */
+        cmuClock_USART0,                    /* CMU register */
+        USART0_RX_IRQn                      /* IRQ base channel */
+    },
+    {
+        USART1,                             /* device */
+        GPIO_PIN(PC, 6),                    /* RX pin */
+        GPIO_PIN(PC, 7),                    /* TX pin */
+        USART_ROUTELOC0_RXLOC_LOC11 |
+            USART_ROUTELOC0_TXLOC_LOC11,    /* AF location */
+        cmuClock_USART1,                    /* CMU register */
+        USART1_RX_IRQn                      /* IRQ base channel */
+    },
+    {
+        LEUART0,                            /* device */
```

Comment:
here and in all other places of this file: please use named struct identifiers, they are much nicer to read and less prone to errors, e.g. `     .dev = LEUART0,`

Reply:
I will also remove the comments behind them, because `.dev = LEUART0 /* device */` is a bit too much :-P